public_html/script/src/drawable/root-drawable.js

```diff
@@ -0,0 +1,68 @@
+/*
+    A drawable to be used as the root drawable (directly on the canvas)
+*/
+'use strict';
+
+import {Drawable} from './drawable';
+import {setCanvas} from '../canvas';
+import {GridManager} from '../grid+draggable/grid-manager';
```

Comment:
Nothing wrong with small folders, and in other applications grids can be used for things that aren't draggables. They'll probably be used with the character movement too, but that's not by dragging. 

The other character to use is another folder...